src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -378,4 +388,116 @@ public InputStream body() throws IOException {
+         * @throws IOException If fails
+         * @todo #252:30min/DEV We should expect instances of Request as dispositions
+         *  Dispositions should be multipart request body holder.
+         *  Ctor. will be called like this.
+         *  {@code
+         *  new RqMultipart.Fake(
+         *  new RqFake("POST", "/upload-a-file"),
+         *  new RqWithHeader(
+         *  new RqFake("", "", "440 Wolfe Rd, CA 94085"),
+         *  "Content-Disposition", "form-data; name=\"address\""
+         *  )
+         *  );
+         *  }
+         */
+        @SuppressWarnings("PMD.UnusedFormalParameter")
+        public Fake(final Request req, final Request... dispositions)
+            throws IOException {
+            this.fake = new RqMultipart.Base(new RqFake());
+        }
+        /**
+         * Fake ctor.
+         * @param req Fake request header holder
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         */
+        public Fake(final Request req, final String... dispositions)
+            throws IOException {
+            this.fake = new RqMultipart.Base(
+                new RqSimple(
+                    new RqWithHeader(
+                        req,
+                        // @checkstyle LineLengthCheck (1 line)
+                        RqMultipart.CONTENT_TYPE, "multipart/form-data; boundary=AaB02x"
+                    ).head(),
+                    RqMultipart.Fake.fakeBody(dispositions)
+                )
+            );
+        }
+        /**
+         * Fake ctor.
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         */
+        public Fake(final String... dispositions) throws IOException {
+            this(new RqFake(), dispositions);
+        }
+        @Override
+        public Iterable<Request> part(final CharSequence name) {
+            return this.fake.part(name);
+        }
+        @Override
+        public Iterable<String> names() {
+            return this.fake.names();
+        }
+        @Override
+        public Iterable<String> head() throws IOException {
+            return this.fake.head();
+        }
+        @Override
+        public InputStream body() throws IOException {
+            return this.fake.body();
+        }
+        /**
+         * Fake body creator.
+         * @param dispositions Fake request body parts
+         * @return InputStream of given dispositions
+         */
+        private static InputStream fakeBody(final String... dispositions) {
+            final String boundary = "AaB02x";
+            final Collection<String> parts = new LinkedList<String>();
+            for (final String disposition : dispositions) {
+                parts.add(String.format("--%s", boundary));
+                parts.add(disposition);
+            }
+            parts.add("Content-Transfer-Encoding: utf-8");
+            parts.add(String.format("--%s--", boundary));
```

Comment:
looks like a duplication of the line above. something is wrong with this method. also, I don't think we need that `join()` below. we can easily implement all this inside one method